tools/toolchain/dbuild

```diff
@@ -90,11 +91,19 @@ elif [[ "$1" = -* ]]; then
                 if [[ -z "$image" ]]; then
                     die "Expected docker image identifier to follow the --image option"
                 fi
-                if ! docker image inspect "$image" >/dev/null && ! docker image pull "$image"; then
+                if ! $tool image inspect "$image" >/dev/null && ! $tool image pull "$image"; then
                     die
                 fi
                 continue
                 ;;
+            --tool)
```

Comment:
Let's have @avikivity's feedback on the name of this command line option, but otherwise looks good to me.

Reply:
"tool" is a very generic name. Why don't we call it $docker, --docker, since that's what it is?

Reply:
maybe i will change it to `dbuild_tool` ?